### DISCUSSION TITLE
#8764 not solved try to response. full node loading continually

### DISCUSSION THREAD

**Discussion by Nagendra129** [Chia-Network/chia-blockchain#8773] [Support]
#8764 not solved try to response. full node loading continually...... Other peers not visible 
Mail id   :    nagendra129@gmail.com

**Comment by BrandtH22** [comment]
@Nagendra129 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
